src/cephadm/cephadm

```diff
@@ -2749,6 +2743,18 @@ def get_ceph_volume_container(ctx: CephadmContext,
     )
 
 
+def set_pids_limit_unlimited(ctx: CephadmContext, container_args: List[str]) -> List[str]:
+    # set containers pid-limits to unlimited rather than defaults (Docker 4096 / Podman 2048)
+    # Useful for daemons like iscsi where the default pid-limit limits the number of luns
```

Comment:
```suggestion
    # set container's pids-limit to unlimited rather than default (Docker 4096 / Podman 2048)
    # Useful for daemons like iscsi where the default pids-limit limits the number of luns
```